### PR TITLE
feat(PT-3255): update approval on bid execution

### PR DESCRIPTION
**Pull request by dmitry-saraykin**

### CODE REVIEW

File: erc20/listener/src/main/kotlin/com/rarible/protocol/erc20/listener/listener/OrderActivityEventHandler.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        val activity = event.activity
+        if (activity is OrderActivityMatchDto && activity.type == OrderActivityMatchDto.Type.ACCEPT_BID) {
+            val sideDto = if (activity.left.type == OrderActivityMatchSideDto.Type.BID) {
+                activity.left
+            } else {
+                activity.right
+            }
+            val assetType = sideDto.asset.assetType
+            if (assetType is Erc20AssetTypeDto) {
+                logger.info("Will recalculate erc20 balance")
```
Comment (yakischik) [inline erc20/listener/src/main/kotlin/com/rarible/protocol/erc20/listener/listener/OrderActivityEventHandler.kt:31]: Provide more info in logs - which balance, from what order etc

File: erc20/listener/src/main/kotlin/com/rarible/protocol/erc20/listener/listener/OrderActivityEventHandler.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class OrderActivityEventHandler(
+    private val erc20AllowanceService: Erc20AllowanceService,
+) : RaribleKafkaEventHandler<EthActivityEventDto> {
+
+    override suspend fun handle(event: EthActivityEventDto) {
+        logger.info("Handle event: $event")
```
Comment (yakischik) [inline erc20/listener/src/main/kotlin/com/rarible/protocol/erc20/listener/listener/OrderActivityEventHandler.kt:21]: I think there is no sense to track all events, only those we are interested in

Comment (dmitry-saraykin) [reply erc20/listener/src/main/kotlin/com/rarible/protocol/erc20/listener/listener/OrderActivityEventHandler.kt:21]: how do we track if something is missed?

Comment (dmitry-saraykin) [reply erc20/listener/src/main/kotlin/com/rarible/protocol/erc20/listener/listener/OrderActivityEventHandler.kt]: reduced amount of logs

**Review by yakischik — APPROVED** [review]

**Pull request merged**